x-pack/plugins/upgrade_assistant/common/types.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Legacy } from 'kibana';
+
+export const UPGRADE_ASSISTANT_TYPE = 'upgrade-assistant';
+export const UPGRADE_ASSISTANT_DOC_ID = 'upgrade-assistant';
```

Comment:
Similar to the type, I think this ID is too generic. Can we make it more specific?

Reply:
Changed. Thanks @joshdover !